pkg/products/cloudresources/reconciler.go

```diff
@@ -357,8 +357,9 @@ func (r *Reconciler) reconcileCIDRValue(ctx context.Context, client k8sclient.Cl
 		return err
 	}
 
-	if !ok {
-		return nil
+	//!ok means the param wasn't found so we want to default rather than return
+	if !ok || cidrValue == "" {
```

Comment:
is there any possibility the secret has not arrived to the cluster yet? I can't recall what Mike said, is the secret always created? Wondering do we want to allow a little time for the secret to get there from hive. Seems like it should be there as it will be created at the same time as the label to apply the SSS but just wondering if there is a chance we set the default then the secret comes in

Reply:
We should check with Mike. As soon as the `_network` entry is added cro will create the vpc and progress with creating the resources and it cannot be changed after that. I'll ping him in the chat.